## Single Cell Atlas from the whole colony of *Hydractinia symbiolongicarpus*
Pipeline used for the processing and analysis of the *Hydractinia symbiolongicarpus* single cell atlas

### [BiorXiv preprint](https://www.biorxiv.org/content/10.1101/2024.06.18.599157v2)

### [UCSC cell browser](https://cells.ucsc.edu/?ds=hydractinia)
   
### [GEO (scRNA-seq reads): GSE269914](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269914)

## Notebooks contain the necesary code to run: 

	1. Library analyses, preproccesing and initial clustering. 
	2. Compositional analyses of the annotated clusters.
	3. Transcription factor annotation, analysis, Coocurrence matrix and WGCNA pipeline.
	4. Shematrin repeat finder custom code and *Stylophora* metacell score. 

All analyses were run using Jupyter Notebook with Python v3 and R v4.1 in Windows OS 8 and Ubuntu GNU/Linux. All packages used are fully described in the methods section of the manuscript. 
